docs/openhabian-amanda.md

```diff
@@ -564,12 +567,12 @@ You could also move that temporary recovered image file to your Windows PC that
 ### A final word on when things have gone badly wrong...
 
 and your SD card to contain the Amanda database is broken: you don't have to give up.
-Whenever you use a directory as the storage area, openHABian Amanda by default creates a copy of its config and index files (to know what's stored where) in your storage directory once a day (see `/etc/cron.d/amanda`).
-So you can reinstall openHABian including Amanda from scratch and copy back those files. Eventually see `amadmin import` option.
+Whenever you use a directory as the storage area, openHABian Amanda by default creates a copy of its config and index files (to know what's stored where) in your storage directory once a day (see `/etc/cron.d/amanda`). This is usually the amanda-backups directory, rather than the slots directory, and you extract the /var/lib/amanda files from the amanda_data_DATE.tar.gz file you want (probably the latest). You can also extract the /var/log/amanda directory if needed.
+So you can reinstall openHABian including Amanda from scratch and copy back those extracted files. Eventually see `amadmin import` option.
 Even if you fail to recover your index files, you can still access the files in your storage area.
 The `amindex` command can be used to regenerate the database. How to apply unfortunately is  out of scope for this document so please g**gle if needed.
 There's also a manual way: Amanda storage files are tar files of the destination directory or compressed raw copies of partitions, both have an additional 32KB header.
-If you just want to retrieve some files from a partition backup file, you can mount that file. See https://major.io/2010/12/14/mounting-a-raw-partition-file-made-with-dd-or-dd_rescue-in-linux/.
+If you just want to retrieve some files (such as the items, things, rules, sitemaps, icons and the jsondb information written directly by the paperUI or HABpanelUI) from a partition backup file in the slots directory, you can mount/download that file. See https://major.io/2010/12/14/mounting-a-raw-partition-file-made-with-dd-or-dd_rescue-in-linux/.
```

Comment:
... by Paper UI) from ...

The /download addition is bad, it's meant differently. It's not about how to get the file to your box but what to do with it then. You really need to _mount_ it like you mount a disk.

Reply:
my headless rp3 writes backups to a Synology NAS and I use Chrome on my laptop to view the backups. I can see the slot folders and then right click on the file I want and select download from the menu to get it on my laptop to run dd and tar xvf to get the file. What if I change it to

you can mount that file to your openhabian box or perhaps download it from your NAS to another computer to run dd and tar and get the files you need

Reply:
No that still shows a misunderstanding of yours.
It is a raw partition dump and you cannot use dd or tar to extract. The only way to access is to _mount_ the _file_ as if it was a disk device name. It really has nothing to do with downloading thus remove that part please. Downloading you wouldn't either as you need to run amrestore on your box and need access to the Amanda dumps to extract the file from backup. NFS != download

Reply:
The current documentation explains how to get back the files without using amrestore, or any part of amanda. My business used to use physical cartridge tapes for all backup/restore and we would use dd and tar on a daily basis. I was able get back everything I needed from the amanda backups without using amanda. What I am trying to do now is make the documentation better for others. I gather that you do not like the word download, even though that is the menu pick you use with a Synology NAS. I will have another go at the wording and examples plus a little on different scenarios for the physical backup device

Reply:
>The current documentation explains how to get back the files without using amrestore, or any part of amanda.

I assume you refer to the very last 'things gone wrong' section only because under normal conditions, there is absolutely no need to download-transfer-whatever any storage file because that is what amrestore/amrecover takes care of. Users should only ever use amrestore/amrecover except when they cannot because things have gone badly wrong
so I will not accept docs to describe how to bypass that.
To describe how to get that file off your storage in this 'gone wrong' situation depends on the medium, and 
> I gather that you do not like the word download, even though that is the menu pick you use with a Synology NAS.

It's two things. Yes one is that I do not like "download" in this context because this is documentation to apply to *all* Amanda users. They may **or may not** use a NAS, but it could also be an USB stick or a DLT drive, and one would not call that download even on a NAS because that by default is mounted just as it is in plain operations, so it's nothing but a "copy" or direct access.

The real thing though is what I wrote in my previous comment:
you're mixing retrieval of the storage file (what your "download" does) with retrieval of files _from_ the storage file (what mount + dd/tar do).
That's two independent steps you must not mix.

Reply:
Hi Markus, I have made some more changes and also just discovered that you have another PR in process to improve the docs. I now have 3 commits on my PR and trust that there are no conflicts with yours. I think I now have made backup device neutral changes. I installed amanda because I was concerned about another sd card crash. When it happened I looked for guidance from the web on what to do and found that it was not simple. Lots of other people also had difficulties. My solution to get openhabian up and running ASAP was to use the dd plus tar method and am happy amanda had stored the backups for me. If I have another sd card failure I may try amanda restore or just use dd plus tar. What I will do is set up another amanda backup to do weekly or fortnightly img backups of my system. Then when I have failure I will just go straight to get the latest img backup via dd and tar. I will not rebuild a temporary openhabian system just to run amanda restore to get my img file and then do another rebuild with that img file. I assume that you can see that there are cases where the user can decide whether amanda restore is best, or not the best. Hope you are staying safe in this crazy COVID-19 time. Best Regards, Andrew Currie